components/sendEmailForm.tsx

```diff
@@ -0,0 +1,88 @@
+import { Form, Button, Select, Row, Col, message } from 'antd';
+import Text from 'antd/lib/typography/Text';
+import { useState } from 'react';
+
+const { Option } = Select;
+
+export interface ManageFormFields2 {
```

Comment:
should be called something better I don't think it's related to manage form

Reply:
This was just there for testing purposes, will remove.